### ISSUE TITLE
Loss and Epocs

### ISSUE THREAD

**Issue by dronefreak** [iamhankai/ghostnet.pytorch#18]
Hi @iamhankai 

I am attempting to re-train GhostNet (MobileNetV3) on the ImageNet dataset with some changes in activations and fine-tuning hyperparameters. Could you please tell me how many epochs you trained the network for and what was your final loss value? I could not find them in the paper. Thanks.

**Comment by iamhankai** [comment]
refer to https://github.com/huawei-noah/ghostnet/issues/9

**Comment by dronefreak** [comment]
Hi @iamhankai 

Thanks for your response. The issue that you have referenced does not mention the information as requested. I am trying to train GhostNet on ImageNet from scratch with modifications for my research. 
I am very new to Tensorpack usage (and DL in general) so it would be really helpful if I could get the training code for the same.

**Comment by iamhankai** [comment]
400 epochs. The final loss is 1.9572.

**Comment by dronefreak** [comment]
Thanks mate.
